Pod/Classes/BONChain.h

```diff
@@ -78,6 +79,11 @@ typedef BONChain *BONCNonnull (^BONTagComplexStyles)(BONGeneric(NSArray, BONTag
 
 @property (copy, nonatomic, readonly) BONChainBaselineOffset baselineOffset;
 
+/**
+ *  Hyphenation is attempted when the ratio of the text width (as broken without hyphenation) to the width of the line fragment is less than the hyphenation factor. When the paragraph’s hyphenation factor is 0.0, the layout manager’s hyphenation factor is used instead. When both are 0.0, hyphenation is disabled.
```

Comment:
It would be nice to have some guidance on what kinds of numbers to use here. I could say the same for Apple's documentation, but if you have any rules of thumb, that would be helpful.